docs/application/native/guides/multimedia/webrtc.md

```diff
@@ -0,0 +1,447 @@
+    }
+    ```
+    > **Note**
+    >
+    > `webrtc_set_sound_stream_info()` or `webrtc_set_display()` must be called inside of the callback set by `webrtc_set_track_added_cb()` if you want to output the audio or video track from remote peer to local target device's audio device or video display.
+
+2. To get media packet handle which packs the audio or video data from remote peer, use `webrtc_set_encoded_audio_frame_cb()` or `webrtc_set_encoded_video_frame_cb()` function:
+
+    ```c
+    void _encoded_frame_cb(webrtc_h webrtc, webrtc_media_type_e type, unsigned int track_id, media_packet_h packet, void *user_data)
+    {
+        some_app_data_s *data = (some_app_data_s *)user_data;
+
+        if (type == WEBRTC_MEDIA_TYPE_AUDIO) {
+            /* Use media packet - copy it's data or pass it to another API */
+        } else if (type == WEBRTC_MEDIA_TYPE_VIDEO) {
+            /* Use media packet - copy it's data or pass it to another API */
+        }
+
+        /* media packet should be unreferenced after use */
+	    media_packet_unref(packet);
+    }
+
+    void webrtc_func(void)
+    {
+        int ret;
+        webrtc_h webrtc;
+        ...
+        ret = webrtc_set_encoded_audio_frame_cb(webrtc, _encoded_frame_cb, data);
+        ret = webrtc_set_encoded_video_frame_cb(webrtc, _encoded_frame_cb, data);
+        ...
+        ret = webrtc_start(webrtc);
+        ...
+        /* After finishing negotiation, _encoded_frame_cb() could be called if receiving audio/video data from remote peer exists */
+    }
+    ```
+
+3. To render sending audio/video data on local target device, use `webrtc_media_source_set_audio_loopback()` or `webrtc_media_source_set_video_loopback()` function:
+
+    ```c
+    void webrtc_func(some_app_data_s *data)
+    {
+        int ret;
+        webrtc_h webrtc;
+        unsigned int a_src_id, v_src_id;
+        unsigned int a_track_id, v_track_id;
+        ...
+        ret = webrtc_add_media_source(webrtc, WEBRTC_MEDIA_SOURCE_TYPE_MIC, &a_src_id);
+        ret = webrtc_add_media_source(webrtc, WEBRTC_MEDIA_SOURCE_TYPE_CAMERA, &v_src_id);
+
+        /* To set audio source data loopback and to output it to audio device on local target device */
+        ret = webrtc_media_source_set_audio_loopback(webrtc, a_src_id, data->stream_info, &track_id);
+        /* To set video source data loopback and to render it to video overlay, use window id */
+        ret = webrtc_media_source_set_video_loopback(webrtc, v_src_id, WEBRTC_DISPLAY_TYPE_OVERLAY, data->win_id, &track_id);
+        ... or ...
+        /* To set video source data loopback and to render it to EVAS image object */
+        ret = webrtc_media_source_set_video_loopback(webrtc, v_src_id, WEBRTC_DISPLAY_TYPE_EVAS, data->evas_image_object, &track_id);
+        ...
+        ret = webrtc_start(webrtc);
+        /* Do negotiation */
+    }
+    ```
+
+## Related Information
```

Comment:
For second headings, please use sentence style capitalization
Ref: https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#capitalization
```suggestion
## Related information
```